nexus/types/src/deployment/blueprint_display.rs

```diff
@@ -204,16 +242,54 @@ impl fmt::Display for BpSledSubtable {
         for row in &self.rows {
             let prefix = row.state.prefix();
             write!(f, "{prefix:<SUBTABLE_INDENT$}")?;
+            let mut multiline_row = false;
             for (i, (column, width)) in
                 row.columns.iter().zip(&widths).enumerate()
             {
+                let (column, needs_multiline) = match column {
+                    BpSledSubtableColumn::Value(s) => (s.clone(), false),
+                    BpSledSubtableColumn::Diff { before, .. } => {
+                        (format!("{REMOVED_PREFIX} {before}"), true)
+                    }
+                };
+                multiline_row |= needs_multiline;
+
                 if i != 0 {
                     write!(f, "{:<COLUMN_GAP$}{column:<width$}", "")?;
                 } else {
                     write!(f, "{column:<width$}")?;
                 }
             }
             write!(f, "\n")?;
+
+            // Do we need any multiline output?
+            if multiline_row {
```

Comment:
It's not obvious to me how this will scale to multiple multiline rows. We don't need that now so that seems fine, but I'm curious if you have an idea what that will look like?

Reply:
Honestly, i can only see the need for one more row, with a description of what happened. Anything else will be ridiculous clutter. I think we'll scale it by adding another boolean and loop to the rendering function, along with more detail in a column. It's done this way because of the way the rendering is done line by line. Another way to do it would be to compute the output into another intermediate format and then render that form. That really seems like overkill right now for what we likely won't need. I suggest re-evaluating when we have more details about what we need beyond a "reason or multiple reason" lines.